.github/workflows/scrape.yml

```diff
@@ -74,7 +74,7 @@ jobs:
           heroku_api_key: ${{ secrets.HEROKU_API_KEY }}
           heroku_email: ${{ secrets.HEROKU_ACCOUNT }}
           heroku_app_name: ${{ secrets.HEROKU_APP }}
-          command: python manage.py update_index --batch-size=50 --age=1
+          command: python manage.py update_index --batch-size=50 --age=1 && python manage.py populate_person_statistics
```

Comment:
not 100% sure this is the right syntax. is there documentation to refer to?

Reply:
this fine. generally, you find the documentation for a particular action by searching google for that action name "michcio1234/heroku-run" and then seeing what it says. Unfortunately, this is a pretty shittily documented one, but this will work

Reply:
cool good to know